Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn \
 
 EXPOSE 8000
 
-CMD [ "yarn", "webpack-dev-server", "--port", "8000" ]
```

Comment:
This command doesn't actually exist anymore but it was never updated. Probably because the image is never run with this default command but rather a command override is supplied by Docker Compose.